src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -223,26 +231,20 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction UpperT = _library.Add(new LowerUpperTFunction(isLower: false));
-        public static readonly TexlFunction Value = _library.Add(new ValueFunction());
-        public static readonly TexlFunction Value_UO = _library.Add(new ValueFunction_UO());
-        public static readonly TexlFunction VarP = _library.Add(new VarPFunction());
-        public static readonly TexlFunction VarPT = _library.Add(new VarPTableFunction());
-        public static readonly TexlFunction Weekday = _library.Add(new WeekdayFunction());
+        public static readonly TexlFunction UTCNow = _featureGateFunctions.Add(new UTCNowFunction());
+        public static readonly TexlFunction UTCToday = _featureGateFunctions.Add(new UTCTodayFunction());
+        public static readonly TexlFunction Value = _library.Add(new ValueFunction());
+        public static readonly TexlFunction Value_UO = _library.Add(new ValueFunction_UO());
+        public static readonly TexlFunction VarP = _library.Add(new VarPFunction());
+        public static readonly TexlFunction VarPT = _library.Add(new VarPTableFunction());
+        public static readonly TexlFunction Weekday = _library.Add(new WeekdayFunction());
         public static readonly TexlFunction WeekdaysLong = _library.Add(new WeekdaysLongFunction());
-        public static readonly TexlFunction WeekdaysShort = _library.Add(new WeekdaysShortFunction());
-        public static readonly TexlFunction WeekNum = _library.Add(new WeekNumFunction());
-        public static readonly TexlFunction With = _library.Add(new WithFunction());
+        public static readonly TexlFunction WeekdaysShort = _library.Add(new WeekdaysShortFunction());
+        public static readonly TexlFunction WeekNum = _library.Add(new WeekNumFunction());
+        public static readonly TexlFunction With = _library.Add(new WithFunction());
         public static readonly TexlFunction Year = _library.Add(new YearFunction());
-
-        public static readonly TexlFunction Decimal = _featureGateFunctions.Add(new DecimalFunction());
-        public static readonly TexlFunction Decimal_UO = _featureGateFunctions.Add(new DecimalFunction_UO());
-        public static readonly TexlFunction EDate = _featureGateFunctions.Add(new EDateFunction());
-        public static readonly TexlFunction EOMonth = _featureGateFunctions.Add(new EOMonthFunction());
-        public static readonly TexlFunction Float = _featureGateFunctions.Add(new FloatFunction());
-        public static readonly TexlFunction Float_UO = _featureGateFunctions.Add(new FloatFunction_UO());
-        public static readonly TexlFunction IsUTCToday = _featureGateFunctions.Add(new IsUTCTodayFunction());
-        public static readonly TexlFunction UTCNow = _featureGateFunctions.Add(new UTCNowFunction());
-        public static readonly TexlFunction UTCToday = _featureGateFunctions.Add(new UTCTodayFunction());
+
+        // Don't add new functions here, follow alpha order

```

Comment:
>// Don't add new functions here, follow alpha order [](http://example.com/codeflow?start=8&length=51)

It's actually useful to have the functions that are not part of the core separate from the core ones. If they are intermingled with the other ones it is hard to spot quickly which is which.

Reply:
The ones in the _library (which are exposed in the `BuiltinFunctionsLibrary` property) are the core ones. The ones in `_featureGateFunctions` are not present in all platforms.